src/portal/src/i18n/lang/en-us-lang.json

```diff
@@ -1028,6 +1029,41 @@
         "IN_PROGRESS": "Search...",
         "BACK": "Back"
     },
+    "SBOM": {
+        "CHART": {
+            "SCANNING_TIME": "Scan completed time:",
+            "SCANNING_PERCENT": "Scan completed percent:",
```

Comment:
Percentage of completed scan.
or 
Scan Completion in percent
or
Scan progress